.goreleaser.yml

```diff
@@ -37,15 +37,16 @@ archives:
       - LICENSE
 
 dockers:
-  - goos: linux
+  - use: buildx
+    goos: linux
     goarch: amd64
     ids:
     - listmonk
     image_templates:
-    - "listmonk/listmonk:latest"
```

Comment:
Does this mean that a `docker pull listmonk/listmonk:latest` will no longer work or will the `-amd64` bit resolve automatically?

Reply:
docker automatically resolves this based on the system's architecture. so running `docker pull listmonk/listmonk:latest` on an `amd64` machine will pull the `amd64` image, whereas on `arm` it'll automatically pull the `arm` version of the image.

Reply:
source: https://docs.docker.com/build/building/multi-platform/#building-multi-platform-images